pkgs/by-name/si/simde/package.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "simde";
+  version = "0.7.6";
+
+  src = fetchFromGitHub {
+    owner = "simd-everywhere";
+    repo = "simde";
+    rev = "v${version}";
+    hash = "sha256-pj+zaD5o9XYkTavezcQFzM6ao0IdQP1zjP9L4vcCyEY=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/include
+    cp -a ${pname} $out/include
+
+    install -Dt $out/share/doc/${pname} README.md
+
+    runHook postInstall
+  '';
```

Comment:
This way of packaging isn't entirely clear to me. Upstream seems like it's using meson as their build tool. Maybe it might make sense to add `meson` to `nativeBuildInputs`?

Reply:
Honestly, I'm not that familar with meson and I didn't see a `Makefile` or similar and combined with the fedora spec file manually copying the files I assumed that was the most straightforward path. Thanks for the suggestion it bundled up nicely.